File: HomeWorks/CPP/Power without Mult-operator.cpp
#include<iostream>
using namespace std;

int main()
{
    int num, power, ans=1;
    cout<<"Enter number : ";
    cin>>num;
    cout<<"Enter power : ";
    cin>>power;

    if( (power == 0) || (num == 1))
    {
        cout<<"\nD-Bug : power==0\n\n";
        cout<<"Answer = 1\n\n";
    }
    else if(power == 1)
    {
        cout<<"\nD-Bug : power==1\n\n";
        cout<<"Answer = "<<num<<"\n\n";
    }
    else
    {
        cout<<"\nD-Bug : ";
        while(power>0)
        {
            cout<<"\n  num : "<<num<<"\n  power : "<<power<<"\n  ans : "<<ans<<endl;
            int i_ans = ans;
            int sum = 0;
            while(i_ans>0)
            {
                sum += num;
                i_ans--;
            }
            ans=sum;
            power--;
        }
        cout<<"\n  num : "<<num<<"\n  power : "<<power<<"\n  ans : "<<ans<<"\nD-Bug."<<endl;
        cout<<"\n\nAnswer = "<<ans<<endl;
    }
    return 0;
}
//MadMad_43

File: HomeWorks/PY/Power without Mult-operator.py
num = int(input("please enter number : "))
power = int(input("please enter power : "))
ans=1

if( (power == 0) or (num == 1)):
    print("answer = 1")
elif(power == 1):
    print("answer = ", num)
else:
    while(power>0):
        i_ans = ans
        sum=0
        while(i_ans>0):
            sum+=num
            i_ans-=1
        ans=sum
        power-=1
    print("answer = ", ans)
#MadMad_19